pypiper/manager.py

```diff
@@ -1693,18 +1694,28 @@ def clean_add(self, regex, conditional=False, manual=False):
             # Override the user-provided option and force manual cleanup.
             manual = True
 
+        if not self.clean_initialized:
+            # Make cleanup files relative to the cleanup script in case the result folder moves.
+            with open(self.cleanup_file, "a") as myfile:
```

Comment:
It makes sense to do append mode, but what happens if the same cleaning commands are written to the cleanup file? Say, if the pipeline shuts down. Is it possible to get dual writes, or is the cleanup script always removed?